src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -278,7 +286,7 @@ int main(int argc, const char* argv[]) {
   passRunner.setDebug(options.debug);
   passRunner.setDebugInfo(debugInfo);
 
-  if (checkStackOverflow && !sideModule) {
+  if (checkStackOverflow) {
```

Comment:
Can side modules now support stack checks when they couldn't before?

Reply:
Without `--mutable-sp` the side module is forced to manipulate SP indirectly via the `stackSave`/`stackRestore` functions that live in the main module.  

So we go away without running this pass on the side module previously.